lib/ohai/plugins/filesystem.rb

```diff
@@ -154,11 +154,21 @@ def run_with_check(bin, &block)
 
     # Grab filesystem data from df
     run_with_check("df") do
-      so = shell_out("df -P")
+      # df runs statfs across all filesystems including remote ones. When run
+      # at scale, Ohai adds extra load, simply to obtain block / inode usage
+      # free information. This is rarely useful. Turn on this option to
+      # disable remote filesystem enumeration in df. Note that device +
+      # mountpoint will still be obtained from mount and /proc/mounts.
+      params = if Ohai.config[:plugin][:filesystem][:no_remote]
+        ' --local'
+      else
+        ''
+      end
+      so = shell_out("df -P#{params}")
       fs.merge!(parse_common_df(so.stdout))
 
       # Grab filesystem inode data from df
-      so = shell_out("df -iP")
+      so = shell_out("df -iP#{local}")
```

Comment:
did you mean `params` ?

Reply:
yeah, not enough coffee yesterday it seems.